src/gui/folderstatusmodel.cpp

```diff
@@ -589,6 +589,13 @@ void FolderStatusModel::fetchMore(const QModelIndex &parent)
     QTimer::singleShot(1000, this, &FolderStatusModel::slotShowFetchProgress);
 }
 
+void FolderStatusModel::resetAndFetch(const QModelIndex &parent)
+{
+    auto info = infoForIndex(parent);
+    info->_fetched = false;
```

Comment:
This would still not re-fetch if there was already a fetch in progress  
(info->_fetchingJob)

I think the best is to simply call `resetSubs` here.
Calling it twice does not hurt, and it is even more explicit of what it does.

Reply:
ok